File: examples/fibonacci.py
def fib(n: int) -> int:
    if n <= 1:
        return n
    a: int = 0
    b: int = 1
    i: int = 2
    while i <= n:
        temp: int = a + b
        a = b
        b = temp
        i += 1
    return b

def main() -> int:
    result: int = fib(10)
    print(result)
    return 0

File: examples/fibonacci.rs
fn fib(n: i32) -> i32 {
    if n <= 1 {
        return n;
    }
    let a: i32 = 0;
    let b: i32 = 1;
    let i: i32 = 2;
    while i <= n {
        let temp: i32 = a + b;
        a = b;
        b = temp;
        i += 1;
    }
    return b;
}

fn main() -> i32 {
    let result: i32 = fib(10);
    println!("{}", result);
    return 0;
}
